### PR TITLE
Update to DV 6.3.8 based on CR1

### PR DESCRIPTION
**Pull request by jrenaat**
Signed-off-by: Jan Schatteman <jschatte@redhat.com>

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -24,8 +24,8 @@ sources:
       md5: f88f9106745f913c9e2e158cd28eebf7
       # TODO: update link to point to the correct file
```
Comment (goldmann) [inline image.yaml]: Can we remove this TODO? Is it still required?

Comment (jrenaat) [reply image.yaml]: I don't know, you tell me?
You're the one who put it there, according to git history ... ;)

Comment (goldmann) [reply image.yaml]: But you are owning now the source :)

I checked this. looks good, comment can be removed.

**Review by goldmann — APPROVED** [review]
LGTM

**Pull request merged**